src/discussions/tours/data/selectors.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/prefer-default-export
+export const notRespondedFilterTour = ({ tours }) => {
+  // This function filters the tours list in the state by the tour_name 'not_responded_filter'
+  // and returns the filtered list. This can be useful for displaying only the 'not_responded_filter'
+  // tours to the user, for example in a list or table.
+  if (!tours) {
+    return {};
+  }
+  const response = tours.tours.find(tour => tour.tourName === 'not_responded_filter');
```

Comment:
You can also create a new selector called `selectTours => (state) => state.tours.tours` and use `selectTours` selector in this selector. which is make it cleaner and you can use `selectTours` anywhere to get the tour data.